examples/nlp/text_normalization_as_tagging/utils/eval_per_class.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+This script can be used to compare the inference output of Thutmose tagger with multi_reference file.
+The additional report is stored to a separate file for each semiotic class.
+
+USAGE Example:
+  python eval.py \
```

Comment:
wrong call. could eval.py and eval_per_class.py be combined into a single file and run one after another as they use the same inputs?